python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -499,28 +600,27 @@ def is_started(self):
         return not isinstance(self.worker_group, InactiveWorkerGroup)
 
     @property
-    def latest_run_dir(self) -> Optional[Path]:
-        """Path to the latest run directory."""
-        if self._run_id > 0:
-            run_dir = Path(f"run_{self._run_id:03d}")
-            return construct_path(run_dir, self.logdir)
-        else:
-            return None
+    def latest_run_dir(self):
+        return self.checkpoint_manager.latest_run_dir
 
     @property
     def latest_checkpoint_dir(self) -> Optional[Path]:
         """Path to the latest checkpoint directory."""
-        checkpoint_dir = Path("checkpoints")
-        return construct_path(checkpoint_dir, self.latest_run_dir)
+        return self.checkpoint_manager.latest_checkpoint_dir
 
     @property
     def latest_checkpoint_path(self) -> Optional[Path]:
         """Path to the latest persisted checkpoint."""
-        if self._latest_checkpoint_id > 0:
-            checkpoint_file = f"checkpoint_{self._latest_checkpoint_id:06d}"
-            return self.latest_checkpoint_dir.joinpath(checkpoint_file)
-        else:
-            return None
+        return self.checkpoint_manager.latest_checkpoint_path
+
+    @property
+    def latest_checkpoint_id(self):
+        return self.checkpoint_manager._latest_checkpoint_id
```

Comment:
We can return `None` if there have not been any checkpoints:
```suggestion
    @property
    def latest_checkpoint_id(self) -> Optional[int]:
        if self.checkpoint_manager._latest_checkpoint_id > 0:
           return self.checkpoint_manager._latest_checkpoint_id
        else:
           return None
```